packages/manager/apps/dedicated/package.json

```diff
@@ -24,6 +24,7 @@
     "@ovh-ux/manager-enterprise-cloud-database": "^0.1.7",
     "@ovh-ux/manager-error-page": "^1.0.0",
     "@ovh-ux/manager-exchange": "^1.0.0",
+    "@ovh-ux/manager-iplb": "^0.0.0",
```

Comment:
```suggestion
    "@ovh-ux/manager-iplb": "^0.0.0 || ^1.0.0",
```

Reply:
vps and veeam uses ^0.1.0

Reply:
@radireddy any features inside `^0.0.0` semver range will be seen as a Breaking Change so that's why we suggest at the initialization of the module to define this following range (from @marie-j's comment).

Reply:
@antleblanc and @marie-j 
I have changed it to ^0.1.0 
Is it fine for you?

Reply:
@radireddy please keep the @marie-j's suggestion.

Reply:
@radireddy this won't change anything. 
Every minor version for a `0.x.x` major is considered as breaking (due to the fact that a `0.x.x` version is not considered as stable yet I suppose) 
Due to our automatic release & bump if the module is in version `0.1.0` and is released into a `0.2.0` (because of a feature commit), the `^0.1.0` range will only take `0.1.x` versions and the `0.2.0` version won't match.
That's why you need to specify a `^0.0.0 || ^0.1.0` range so your build doesn't break

Reply:
changed to ^0.0.0 || ^0.1.0